.github/workflows/docker.yml

```diff
@@ -72,31 +81,42 @@ jobs:
         uses: docker/metadata-action@v5
         with:
           images: |
-            ${{ secrets.DOCKER_HUB_USERNAME }}/${{ matrix.platform.ruby_target }}
+            ${{ secrets.DOCKER_HUB_USERNAME }}/${{ env.ruby_platform }}
           flavor: |
             latest=auto
+          sep-tags: ','
           tags: |
             type=ref,event=branch
             type=ref,event=pr
             type=semver,pattern={{version}}
             type=sha,format=long
           labels:
-            org.opencontainers.image.description=Image for building native Rust extensions for Ruby on ${{ matrix.platform.ruby_target }}
+            org.opencontainers.image.description=Image for building native Rust extensions for Ruby on ${{ env.ruby_platform }}
             org.opencontainers.image.vendor=oxidize-rb
-            org.oxidize-rb.ruby.platform=${{ matrix.platform.ruby_target }}
+            org.oxidize-rb.ruby.platform=${{ env.ruby_platform }}
+
+      - name: Prepare tags
+        id: prepare_tags
+        run: |
+          if [ -n "${{ env.aliases }}" ]; then
+            combined_tags="${{ steps.meta.outputs.tags }},${{ env.aliases }}"
```

Comment:
Hmm, actually, these aliases don't have the suffix from the meta: https://github.com/oxidize-rb/rb-sys/actions/runs/12818599166/job/35744461501?pr=494